src/modules/watch/router.js

```diff
@@ -25,7 +25,7 @@ module.exports = async (server, opts) => {
     }
   })
 
-  server.put('/:id/targets', { schema: schema.updateTargets }, async (req, res) => {
+  server.put('/:id/targets/data', { schema: schema.updateTargetsData }, async (req, res) => {
```

Comment:
Bởi vì targets đều có ID riêng, việc cung cấp watch ID ở đây là không cần thiết.
Và t nghĩ để cho tiện cho chúng ta thì chắc chỉ cho phép sửa từng target một. Phía client như crawler nhét vào vòng lặp và gửi 2-3 requests liền nếu cần thôi.
 
```suggestion
  server.put('/targets/:id/data', { schema: schema.updateTargetData }, async (req, res) => {
```

Reply:
À còn vấn đề quan trọng nữa, route cũ này còn có tác dụng cập nhật `checkedAt`, nếu body là mảng rỗng. Khi đổi sang route mới thì không thể hỗ trợ nữa, ông tìm cách giải quyết nhé.